docs/CNV/howtogcnv.md

```diff
@@ -0,0 +1,707 @@
+
+Here is one approach to convert gCNV VCF results to SEG format data. These command works in Unix. For Linux, modification will be necessary, e.g. `zcat` replaces `gzcat`.
+
+**[1] VariantsToTable to subset and columnize annotations**
+
+```
+gatk VariantsToTable \
+-V genotyped-segments-case-twelve-vs-cohort23.vcf.gz \
+-F CHROM -F POS -F END -GF NP -GF CN \
+-O genotyped-segments-case-twelve-vs-cohort23.table.txt
+```
+
+**[2] Unix shell commands to convert to SEG format data**
+
+This involves adding a first column with the sample name.
+
+```
+sampleName=$(gzcat genotyped-segments-case-twelve-vs-cohort23.vcf.gz | grep -v '##' | head -n1 | cut -f10)
+awk -v sampleName=$sampleName 'BEGIN {FS=OFS="\t"} {print sampleName, $0}' genotyped-segments-case-twelve-vs-cohort23.table.txt > ${i}.seg; head ${i}.seg
+```
+
+**Combine [1] and [2] into a for-loop to process multiple gCNV VCFs into SEGs**
+
+
+```
+for i in `ls genotyped-segments*.vcf.gz`; 
+do sampleName=$(gzcat $i | grep -v '##' | head -n1 | cut -f10); 
+gatk4100 VariantsToTable -V ${i} -F CHROM -F POS -F END -GF NP -GF CN -verbosity ERROR -O ${i}.table.txt; 
+awk -v sampleName=$sampleName 'BEGIN {FS=OFS="\t"} {print sampleName, $0}' ${i}.table.txt > ${i}.seg; head ${i}.seg; 
+done
+```
+
+<a name="7"></a>
+[back to top](#top)
+
+---
+## 7.	Some additional discussion
+
+If research aims focus on rare gCNV events only, e.g. non-multialleleic CNV singletons for a sample compared to a cohort, it is worth looking into the [GATK4 ModelSegments CNV workflow](https://software.broadinstitute.org/gatk/documentation/article?id=11682), which is sensitive to fractional changes and runs amazingly quickly. Note the ModelSegments CNV workflow is designed for somatic CNA detection. Below is one possible outcome from running ModelSegments CNV on NA19017 against a PoN made from the remaining 23 samples.
+
+[<img src="images-gcnv/gcnv_modelsegments_NA19017.modeled_20190322_sooheelee.png" align="center" width="" />](images-gcnv/gcnv_modelsegments_NA19017.modeled_20190322_sooheelee.png) 
+
+Before moving to the final discussion subsections, my thanks to [@slee](https://gatkforums.broadinstitute.org/gatk/profile/slee), no relation, for guidance and insights on the gCNV workflow.
+
+<a name="7.1"></a>
+### ☞ 7.1 How do gCNV calls correlate with short variant calls?
+[<img src="images-gcnv/gcnv_igv_three_modes_filteredshortvariants_20190321_sooheelee.png" align="right" width="480" />](images-gcnv/gcnv_igv_three_modes_filteredshortvariants_20190321_sooheelee.png) If we examine regions the short variant discovery workflow filters alongside our CNV events (top track), we see an interesting correlation. My thanks to [@bshifaw](https://gatkforums.broadinstitute.org/gatk/profile/bshifaw) for graceously agreeing to prepare the short variant callset. For the particular locus to the right, it appears short variants coinciding with the copy number event are mostly filtered. What could explain this coincidence?
```

Comment:
I think 7.1 and 7.2 could probably be trimmed, they are a bit tangential.  7.3 should be useful once fleshed out.

Reply:
I've cut section 7 altogether from the main tutorial. I hit the Vanilla limit in how long posts can be. I can put 7.1 and 7.2 to a blogpost or other article. I moved mention of the notebooks (7.3 bullet points) to the top of the tutorial (in the introduction). The notebooks are now at:

- https://gatkforums.broadinstitute.org/gatk/discussion/11685/notebook-concordance-of-na19017-chr20-gcnv-calls/p1?new=1
- https://gatkforums.broadinstitute.org/gatk/discussion/11686/notebook-correlate-gcnv-callset-metrics-and-annotations/p1?new=1

They condense the analyses in the multiple notebooks that I had been developing and sharing with you in our meetings so hopefully, the content should be familiar to you @samuelklee.